oeis.py

```diff
@@ -382,6 +382,32 @@ def A000108(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(int(r))
 
     return sequence
+def fibonacci(n):
```

Comment:
As PEP8 tells, please insert two blank lines between functions (or use black to add them).